go/scripts/shipit

```diff
@@ -51,15 +51,17 @@ godep restore
 go get github.com/mitchellh/gox
 popd
 
+if [[ -n ${STATIC_BINARY} && ${STATIC_BINARY} != "false" ]]; then
+	export CGO_ENABLED=0
+fi
 
 pushd $REPO_ROOT
 mkdir artifacts
-gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" .
-go build -ldflags="-X main.Version=${VERSION}" .
+gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" ./...
```

Comment:
Unsure, it was needed for spruce, since it's binary is in a sub package. I figured when we got to pipelining shield on the go side, we would address it then.

Sent from my iPhone

> On Jun 16, 2016, at 8:29 AM, James Hunt <notifications@github.com> wrote:
> 
> In go/scripts/shipit:
> 
> >  
> >  pushd $REPO_ROOT
> >  mkdir artifacts
> > -gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" .
> > -go build -ldflags="-X main.Version=${VERSION}" .
> > +gox -osarch="${TARGETS}" --output="artifacts/${BINARY}-{{.OS}}-{{.Arch}}" -ldflags="-X main.Version=${VERSION}" ./...
> Is the ./... going to work for multi-binary repos like SHIELD, or is gox going to complain about many binaries and one output?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>